testsuite/integration/clustering/src/test/resources/infinispan.xml

```diff
@@ -47,27 +57,21 @@
         </socket-bindings>
 
         <security>
+            <credential-stores>
+                <credential-store name="credentials" path="credentials.pfx">
+                    <clear-text-credential clear-text="secret"/>
+                </credential-store>
+            </credential-stores>
             <security-realms>
                 <security-realm name="default">
-                    <!-- Uncomment to enable TLS on the realm -->
-                    <!-- server-identities>
-                       <ssl>
-                          <keystore path="application.keystore" relative-to="infinispan.server.config.path"
-                                    keystore-password="password" alias="server" key-password="password"
-                                    generate-self-signed-certificate-host="localhost"/>
-                       </ssl>
-                    </server-identities-->
                     <properties-realm groups-attribute="Roles">
-                        <user-properties path="users.properties" relative-to="infinispan.server.config.path" plain-text="true"/>
-                        <group-properties path="groups.properties" relative-to="infinispan.server.config.path"/>
+                        <user-properties path="users.properties"/>
+                        <group-properties path="groups.properties"/>
                     </properties-realm>
                 </security-realm>
             </security-realms>
         </security>
 
-        <endpoints socket-binding="default">
-            <hotrod-connector name="hotrod"/>
-            <rest-connector name="rest"/>
-        </endpoints>
+        <endpoints socket-binding="default" security-realm="default"/>
```

Comment:
Why were the connectors removed?

Reply:
Because that's a typical infinispan server configuration. Declaring an empty <endpoints/> element implicitly enables the Hot Rod and REST connectors.

Reply:
s/typical/default

Reply:
OK - that's a strange choice.

Reply:
Not mine, theirs! I try to keep the infinispan config as close to their defaults as possible and this is their default now.